File: 백준/Bronze/3052. 나머지/나머지.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int[] arr = new int[10];
        int cnt = 0, res = 0;
        for(int i = 0; i < 10; i++){
            arr[i] = in.nextInt()%42;
            for(int j = 0; j < i; j++)
                if(arr[j] != arr[i])
                    cnt++;
            if(cnt == i)
                res++;
            cnt = 0;
        }
        System.out.print(res);
    }
}

File: 백준/Bronze/3052. 나머지/나머지.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var arr = IntArray(10,{0})
    var cnt = 0; var res = 0;
    for(i in 0..9){
        arr[i] = sc.nextInt()%42
        for(j in 0..(i-1)){
            if(arr[j] != arr[i])
            cnt++
        }
        if(cnt == i)
        res++
        cnt = 0
    }
    print(res)
}